server/configuration.go

```diff
@@ -45,7 +45,7 @@ type GlobalConfiguration struct {
 	CheckNewVersion           bool                    `description:"Periodically check if a new version has been released"`
 	AccessLogsFile            string                  `description:"(Deprecated) Access logs file"` // Deprecated
 	AccessLog                 *types.AccessLog        `description:"Access log settings"`
-	TraefikLogsFile           string                  `description:"Traefik logs file"`
+	TraefikLogsFile           string                  `description:"Traefik logs file. Stdout is used when omitted or empty"`
```

Comment:
I added it here by intend, as this is actually the behaviour of the `TraefikLogsFile` and as it wasn't documented yet, I added this comment here as well. Nevertheless, I also extended the documentation in `AccessLog.FilePath`. Good point.